integration-tests/main/src/main/java/io/quarkus/it/corestuff/ImageIOSupport.java

```diff
@@ -14,9 +12,10 @@
 
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
-        ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
-        BufferedImage image = ImageIO.read(classLoader.getResource("META-INF/resources/1px.png"));
-        resp.getWriter().write(image.getHeight() + "x" + image.getWidth());
+        // Please uncomment this when we upgrade from GraalVM 20.3 to GraalVM 21
+        //        ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+        //        BufferedImage image = ImageIO.read(classLoader.getResource("META-INF/resources/1px.png"));
+        //        resp.getWriter().write(image.getHeight() + "x" + image.getWidth());
```

Comment:
Do you really need to leave this commented out? I thought the `@Disabled` below would be enough?

Reply:
No, it won't. It will still be included in the native image and that's what causes the issue. The issue is at build time not runtime.